mirai-core/src/commonTest/kotlin/network/framework/AbstractRealNetworkHandlerTest.kt

```diff
@@ -133,7 +133,8 @@ internal abstract class AbstractRealNetworkHandlerTest<H : NetworkHandler> : Abs
     open fun createContext(): NetworkHandlerContextImpl =
         NetworkHandlerContextImpl(bot, networkLogger, bot.createNetworkLevelComponents())
 
-    val address: InetSocketAddress = InetSocketAddress.createUnresolved("localhost", 123)
+    val address: InetSocketAddress
+        get() = overrideComponents[ServerList].pollAny().let { InetSocketAddress.createUnresolved(it.host, it.port) }
```

Comment:
`val` 获取的内容不应该改变 (如果所属对象状态没有改变), 考虑更换 `by lazy` 或者转换为 `fun createAddress()`

我**简单考虑**了一下, 情况应该更偏向于后者

```suggestion
    val address: InetSocketAddress by lazy {
        overrideComponents[ServerList].pollAny().let { InetSocketAddress.createUnresolved(it.host, it.port) }
    }
```

Reply:
这里确实应该更换为方法 感谢提醒